docs/data-sources/cache.md

```diff
@@ -0,0 +1,99 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_cache Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned cache information
+---
+
+# encore_cache (Data Source)
```

Comment:
Is the encore name the cache cluster, or the keyspace?

Reply:
hmm, I guess it's the cache cluster as we don't actually name the keyspaces